mne/label.py

```diff
@@ -1874,9 +1874,11 @@ def _cortex_parcellation(subject, n_parcel, hemis, vertices_, graphs,
     return labels
 
 
-def _read_annot_cands(dir_name):
+def _read_annot_cands(dir_name, raise_error=True):
```

Comment:
added this because it should not kill the viz if there are no candidates / the `label` dir is missing